common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -74,3 +99,10 @@ steps:
   - script: node common/scripts/install-run-rush.js docs
     displayName: rush docs
     workingDirectory: ${{ parameters.workingDir }}
+
+  # # Will run if even there is a failure somewhere else in the pipeline.
+  # - template: ../templates/publish-test-results.yaml
+  #   parameters:
+  #     NodeVersion: ${{ parameters.nodeVersion }}
+  # # The publish script identifies any new packages not previously published and tags the build
+  # - template: ../templates/publish.yaml
```

Comment:
Can you move the publish step to the `fast-ci.yaml` after this template runs? We don't need/want to run this on every permutation.

Reply:
Sorry, I may be miss understanding. Can I remove this step fully from core-build, and just run this in fast-ci.yaml. Should I not include it in the PR_Validation_Pipeline.yml